core/src/test/java/org/apache/calcite/test/SqlValidatorTest.java

```diff
@@ -5466,6 +5466,63 @@ private ImmutableList<ImmutableBitSet> cube(ImmutableBitSet... sets) {
         .ok();
   }
 
+  @Test void testSelectJoinUsingCommonColumnCaseSensitive() {
```

Comment:
Add the jira title to the test case. You can refer to other case. For example: `/** Test case for...`.